exporters/otlp/common/src/main/java/io/opentelemetry/exporter/otlp/internal/HexEncodingStringJsonGenerator.java

```diff
@@ -13,14 +13,19 @@
 import io.opentelemetry.api.internal.TemporaryBuffers;
 import java.io.IOException;
 
-final class HexEncodingStringJsonGenerator extends JsonGeneratorDelegate {
+public final class HexEncodingStringJsonGenerator extends JsonGeneratorDelegate {
```

Comment:
Moved to `:exporters:otlp:common` and made `create(..)` public because now both the otlp logging exporter and the otlp http exporter with JSON format both need this special encoding.